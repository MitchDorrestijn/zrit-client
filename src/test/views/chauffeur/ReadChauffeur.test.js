import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import ReadChauffeur from '../../../main/views/chauffeur/ReadChauffeur';

const wrapper = shallow(<ReadChauffeur />);

test('<ReadChauffeur /> should match snapshot', () => {
  expect(toJSON(wrapper)).toMatchSnapshot();
});

test('<ReadChauffeur /> should render one <TextContent> component', () => {
  expect(wrapper.find('TextContent').length).toBe(1);
});

test('<ReadChauffeur /> should render one <ChauffeurTable> component', () => {
  expect(wrapper.find('ChauffeurTable').length).toBe(1);
});

test('<TextContent /> in <ReadChauffeur /> should have a title of Chauffeurs beheren', () => {
  const childWrapper = wrapper.find('TextContent');
  expect(childWrapper.prop('title')).toBe("Chauffeurs beheren");
});
