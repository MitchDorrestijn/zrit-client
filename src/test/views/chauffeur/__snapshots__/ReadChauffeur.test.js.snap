// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReadChauffeur /> should match snapshot 1`] = `
<div>
  <TextContent
    title="Chauffeurs beheren"
  >
    Hieronder kunt u chauffeurs beheren. Druk op het rondje vooraan de tabel om een record te selecteren, vervolgens kunt u het record bewerken door op de knop 'bewerken' te drukken. Om een nieuw item toe te toevoegen drukt u op de knop 'Toevoegen'.
  </TextContent>
  <ChauffeurTable />
</div>
`;
